src/swarm/client/connection/NodeErrorTracker.d

```diff
@@ -196,7 +197,7 @@ public class NodeErrorTracker : INodeConnectionPoolErrorReporter
 
             public bool inc ( time_t now, Weights weights )
             {
-                assert(now >= this.last_time);
+                verify(now >= this.last_time);
```

Comment:
I would use `enforce` here, time values are usually program input.

Reply:
I was on the edge of this (since it's not clear if the time input is coming from the clock, or from the ticks). I'll move to enforce.

Reply:
(meaning I would personally not feed this method with the non-monotonic clock)